File: Results/DER/Translation/translation_output/no_test/codeqwen/codenet/python/java/p02111.java
Here is the Java equivalent of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int input = scanner.nextInt();
        int result = input * 2;
        int minutes = result % 60;
        int hours = result / 60;
        System.out.println(hours + " " + minutes);
    }
}
```

This Java code does the same thing as your Python code. It reads an integer from the user, multiplies it by 2, and then finds the remainder when divided by 60 to get the minutes. It then divides the result by 60 to get the hours. Finally, it prints out the hours and minutes.

File: Results/DER/Translation/translation_output/no_test/codeqwen/codenet/java/python/p02111.py
Here is the Python equivalent of the Java code:

```python
import sys

n = int(sys.stdin.readline())
minute = 12 * 60 * n / 360
print((minute // 60) + " " + str(minute % 60))
```

In this Python code, we use `sys.stdin.readline()` to read the input from the standard input (usually the keyboard). The `int()` function is used to convert the input string to an integer. The `//` operator is used for integer division, which gives the quotient without the remainder. The `%` operator is used for the remainder of the division. The `print()` function is used to output the result.
